neqo-transport/benches/transfer.rs

```diff
@@ -63,14 +63,18 @@ fn benchmark_transfer(c: &mut Criterion, label: &str, seed: &Option<impl AsRef<s
 }
 
 fn benchmark_transfer_variable(c: &mut Criterion) {
-    benchmark_transfer(c, "varying-seeds", &std::env::var("SIMULATION_SEED").ok());
+    benchmark_transfer(
+        c,
+        "varying-seeds",
+        std::env::var("SIMULATION_SEED").ok().as_ref(),
+    );
 }
 
 fn benchmark_transfer_fixed(c: &mut Criterion) {
     benchmark_transfer(
         c,
         "same-seed",
-        &Some("62df6933ba1f543cece01db8f27fb2025529b27f93df39e19f006e1db3b8c843"),
+        Some(&"62df6933ba1f543cece01db8f27fb2025529b27f93df39e19f006e1db3b8c843"),
```

Comment:
So you are passing `&&str` and changing the signature to `&impl AsRef`?  That seems a bit "wat" to me.

`impl AsRef<str>` matches `String` and `&str`.  Why does it need an extra `&`?  That only reduces ergonomics.

Reply:
Clippy wants it